microsoft/testsuites/performance/kvmperf.py

```diff
@@ -78,9 +92,147 @@ def perf_nested_kvm_storage_singledisk(
     def perf_nested_kvm_storage_multidisk(
         self, node: RemoteNode, environment: Environment, variables: Dict[str, Any]
     ) -> None:
-        self._perf_qemu(node, environment, variables)
+        self._storage_perf_qemu(node, environment, variables)
+
+    @TestCaseMetadata(
+        description="""
+        This test case runs ntttcp test on two nested VMs on same L1 guest
+        connected with private bridge
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+    )
+    def perf_nested_kvm_ntttcp_private_bridge(
+        self, node: RemoteNode, environment: Environment, variables: Dict[str, Any]
+    ) -> None:
+        self._ntttcp_perf_qemu(node, environment, variables)
+
+    def _ntttcp_perf_qemu(
+        self,
+        node: RemoteNode,
+        environment: Environment,
+        variables: Dict[str, Any],
+    ) -> None:
+        (
+            nested_image_username,
+            nested_image_password,
+            _,
+            nested_image_url,
+        ) = parse_nested_image_variables(variables)
+
+        # setup bridge and taps
+        node.tools[Ip].setup_bridge(self.BR_NAME, self.BR_ADDR)
+        node.tools[Ip].setup_tap(self.CLIENT_TAP, self.BR_NAME)
+        node.tools[Ip].setup_tap(self.SERVER_TAP, self.BR_NAME)
+
+        # setup client and server
+        client = connect_nested_vm(
+            node,
+            nested_image_username,
+            nested_image_password,
+            self.CLIENT_HOST_FWD_PORT,
+            nested_image_url,
+            image_name=self.CLIENT_IMAGE,
+            nic_model="virtio-net-pci",
+            taps=[self.CLIENT_TAP],
+            name="client",
+        )
+        client.tools[Ip].addr_add(self.NIC_NAME, self.CLIENT_IP_ADDR)
+        client.tools[Ip].up(self.NIC_NAME)
+
+        server = connect_nested_vm(
+            node,
+            nested_image_username,
+            nested_image_password,
+            self.SERVER_HOST_FWD_PORT,
+            nested_image_url,
+            image_name=self.SERVER_IMAGE,
+            stop_existing_vm=False,
+            nic_model="virtio-net-pci",
+            taps=[self.SERVER_TAP],
+            name="server",
+        )
+        server.tools[Ip].addr_add(self.NIC_NAME, self.SERVER_IP_ADDR)
+        server.tools[Ip].up(self.NIC_NAME)
+
+        # run ntttcp test
+        client_ntttcp = client.tools[Ntttcp]
```

Comment:
In my latest PR, I moved the method to the common in performance. Let's use the same method to get consistent results. Let me know, if something is not applicable, we can check how to improve it.